src/ResourceManager/Network/Commands.Network/NetworkProfile/ContainerNetworkInterfaceConfiguration/NewAzureNetworkProfileContainerNetworkInterfaceConfigCommand.cs

```diff
@@ -0,0 +1,47 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Management.Network.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetworkProfileContainerNicConfig", SupportsShouldProcess = true), OutputType(typeof(PSContainerNetworkInterfaceConfiguration))]
+    public partial class NewAzureNetworkProfileContainerNetworkInterfaceConfigCommand : AzureNetworkProfileContainerNetworkInterfaceConfigBase
```

Comment:
where are Add-, Set-, REmove and Get- for this?

The [changes](https://github.com/Azure/azure-powershell/pull/7225/files#diff-22072398bfdba9fdc20e34ccc8051168R52) suggest that the following were added :
``` 
               'Get-AzureRmNetworkProfileContainerNicConfig', 
               'New-AzureRmNetworkProfileContainerNicConfig', 
               'Remove-AzureRmNetworkProfileContainerNicConfig', 
               'Add-AzureRmNetworkProfileContainerNicConfig', 
               'Set-AzureRmNetworkProfileContainerNicConfig', 
```

but I only see New